packages/Webkul/Velocity/src/Resources/assets/sass/app.scss

```diff
@@ -136,6 +136,15 @@ label {
     text-decoration: none;
 }
 
+/*
+    default hover theme
+*/
+.default-hover-text{
```

Comment:
Add a space before the bracket @suraj-webkul

Reply:
done